procrastinate/healthchecks.py

```diff
@@ -0,0 +1,37 @@
+from typing import Dict, Tuple
+
+from psycopg2 import Error
+
+from procrastinate import jobs, sql, store, utils
+from procrastinate.migration import Migrator
+
+
+@utils.add_sync_api
+class HealthCheckRunner:
+    def __init__(self, job_store: store.BaseJobStore):
+        self.job_store = job_store
+
+    async def check_connection_async(self) -> Tuple[bool, str]:
+        try:
+            result = await self.job_store.execute_query_one(
+                query=sql.queries["check_connection"],
+            )
+            return result["check"], "OK"
```

Comment:
`result["check"]` should always be `True` here no?

Reply:
Yes, but I find it less surprising to read when using the result of the query.

Reply:
Also, it leaves several possible changes open without changing that line: For example, having a more complex check in SQL, or defering the actual DB request to the `job_store.__getitem__`.